PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx

```diff
@@ -365,18 +403,86 @@ struct HfElectronSelectionWithTpcEmcal {
           continue;
         }
 
-        isEMcal = true;
-        std::cout << " electron id  in selection" << electronId << std::endl;
-        electronSel(matchTrack.collisionId(), electronId, etaMatchTrack, phiMatchTrack, ptMatchTrack, pMatchTrack, trackRapidity, matchTrack.dcaXY(), matchTrack.dcaZ(), matchTrack.tpcNSigmaEl(), matchTrack.tofNSigmaEl(),
-                    eMatchEmcCluster, etaMatchEmcCluster, phiMatchEmcCluster, m02MatchEmcCluster, m20MatchEmcCluster, cellEmcCluster, timeEmcCluster, deltaEtaMatch, deltaPhiMatch, isEMcal);
+        /////////////////////////////////////////////////////////////            photonic electron          ///////////////////////////////////////////////////////
+
+        isLSElectronFound = 0;
+        isULSElectronFound = 0;
+        bool isLSElectron = 0;
+        bool isULSElectron = 0;
+        float invMassElectron = 0.;
+        float massLike = 0;
+        float massUnLike = 0;
+
+        for (const auto& pTrack : tracks) {
+
+          if (pTrack.globalIndex() == matchTrack.globalIndex())
+            continue;
+
+          // Apply Associated electron Selection
+
+          if (!selAssoTracks(pTrack)) {
+            continue;
+          }
+          Int_t PDGe1 = kElectron;
+          Int_t PDGe2 = kElectron;
+          if (matchTrack.sign() > 0)
+            PDGe1 = kPositron;
+          if (pTrack.sign() > 0)
+            PDGe2 = kPositron;
+          KFPTrack kfpTrack = createKFPTrackFromTrack(matchTrack);
+          KFPTrack kfpAssociatedTrack = createKFPTrackFromTrack(pTrack);
+          KFParticle KFTrack(kfpTrack, PDGe1);
+          KFParticle kfAssociatedTrack(kfpAssociatedTrack, PDGe2);
+          const KFParticle* LcDaughters[2] = {&KFTrack, &kfAssociatedTrack};
```

Comment:
what is LcDaughters here?

Reply:
@rashigupt, as I said you many times please use variable names that make some sense, check other vairables too

Reply:
okay